cvat/apps/engine/mixins.py

```diff
@@ -146,21 +165,21 @@ def upload_data(self, request):
         if one_request_upload or finish_upload:
             return self.upload_finished(request)
         elif start_upload:
-            return Response(status=status.HTTP_202_ACCEPTED)
+            return self.upload_started(request)
         elif tus_request:
-            return self.init_tus_upload(request)
+            return self.init_file_upload(request)
         elif bulk_file_upload:
-            return self.append(request)
+            return self.append_files(request)
         else: # backward compatibility case - no upload headers were found
             return self.upload_finished(request)
 
-    def init_tus_upload(self, request):
+    def init_file_upload(self, request):
```

Comment:
Why rename this? I think the old name is better, it shows that this is handling for the case where the file is uploaded with TUS. Same for `append_tus_chunk`.

Reply:
I wanted to hide the fact that TUS is used from the class API. From my point, it should be an insignificant implementation detail.

Reply:
It can't be an implementation detail, because these methods can only be called with request objects that match the requirements of the TUS protocol. So I think they're pretty closely linked with TUS.

Reply:
Ok, probably you're right.